tests/model_fields/tests.py

```diff
@@ -102,6 +102,13 @@ def test_deconstruct_nested_field(self):
         name, path, args, kwargs = Nested.Field().deconstruct()
         self.assertEqual(path, 'model_fields.tests.Nested.Field')
 
+    def test_field_eq_considers_model(self):
+        """Field instances from abstract models compare different."""
+        self.assertNotEqual(
+            AbstractPersonWithHeight._meta.get_field('mugshot_height'),
+            PersonWithHeight._meta.get_field('mugshot_height'),
```

Comment:
These are special models that are available only when `Pillow` is installed we shouldn't use them here. I would add inline models.

Reply:
Fair enough. Writing the tests was the hardest part for me. I'll check out your pull request, to see if there's more for me to do.